libs/application/templates/children-residence-change/src/fields/Duration/index.tsx

```diff
@@ -10,26 +11,31 @@ import {
   RadioController,
 } from '@island.is/shared/form-fields'
 
+const typeInput = 'durationType'
+const dateInput = 'durationDate'
+
+export const selectDurationInputs = [typeInput, dateInput]
+
 export type ValidAnswers = 'permanent' | 'temporary' | undefined
-const Duration = ({ field, application, error }: CRCFieldBaseProps) => {
-  // TODO: Fix when we fix the schema validation for this field
-  const currentAnswer = application.answers.selectDuration
-    ? (application.answers.selectDuration[0] as ValidAnswers)
-    : undefined
+const Duration = ({ application, errors }: CRCFieldBaseProps) => {
+  const currentAnswer = application.answers?.durationType || undefined
```

Comment:
Isn't this equal to:
`  const currentAnswer = application.answers.durationType` ?